tensorboard/webapp/feature_flag/views/feature_flag_page.ng.html

```diff
@@ -16,16 +16,37 @@
 -->
 
 <div>
-  <table>
-    <ng-container *ngFor="let flag of getFlagKeys();">
+  <div class="message">
+    <h2 class="warning">WARNING: EXPERIMENTAL FEATURES AHEAD!</h2>
+    By enabling these features, you could put the application in an unusable
+    state or expose yourself to untested features or potential bugs.
+  </div>
+  <table class="feature-flag-table">
+    <ng-container *ngFor="let flagStatus of featureFlagStatuses;">
       <tr>
-        <mat-checkbox
-          [checked]="featureFlags[flag]"
-          (change)="flagChanged(flag)"
-          >Enabled</mat-checkbox
-        >
-        {{flag}}
+        <td>
+          <div>{{flagStatus.flag}}</div>
+        </td>
+        <ng-container
+          *ngIf="flagStatus.defaultValue === true || flagStatus.defaultValue === false; then selectBlock else staticBlock"
```

Comment:
I found `flagStatus.defaultValue === true || flagStatus.defaultValue === false` to be a bit amusing but maybe it would be clearer if we instead checked `typeof flagStatus.defaultValue === 'boolean'`